### ISSUE TITLE
Can't launch Spyder on Windows 10

### ISSUE THREAD

**Issue by alexkopytin** [spyder-ide/spyder#5525]
## Description of your problem

**What steps will reproduce the problem?**

I typically open Spyder via the Start Menu, but now, when I try to click on the Spyder icon in the Start Menu, I get no response. I then tried to go directly to the spyder.exe file in the Scripts folder in the directory where Anaconda is installed but I got no response. 

**Please provide any additional information below**

## Versions and main components

* Spyder Version: 3.2.4
* Python Version: 3.6.2
* Qt Version: 5.6.2
* PyQt Version: 5.6.0
* Operating system: Windows 10

## Dependencies

Please go to the menu entry `Help > Optional Dependencies` (or
`Help > Dependencies`), press the button `Copy to clipboard`
and paste the contents below:

**Comment by alexkopytin** [comment]
spyder --reset
solves the problem

**Comment by Belay7** [comment]
I am having a similar problem. When I attempt to run Spyder, I get a pop-up saying that Python has stopped working. I have tried launching from start menu, navigator, and prompt, but I get the same result each time. 

I am running Anaconda Navigator 1.8.7, with Spyder 3.2.8, and I have Python 3.6 installed on the machine.

**Comment by ccordoba12** [comment]
Most probably it's an antivirus original firewall problem. Please deactivate them an try again.

**Comment by Belay7** [comment]
I deactivated both and still had the same problem.  I tried restarting with both still deactivated, and still had the same issue.  Spyder will not start when trying to open from start menu or Anaconda Navigator, and I am still getting the message that python has stopped working when I try launching from prompt.

**Comment by jayrawatrj** [comment]
same problem here.. spyder window is showing loading editorand it never open nor get cloosed. i hav eto close it manualy

**Comment by MChetanSR** [comment]
I too have the same issue. It is stuck at the "loading editor..." phase.  Nor could I launch jupyter notebook. When I run jupyter.exe or jupter-troubleshoot.exe, a window opens and closes in a fraction of second. I had this problem a month ago. So I switched to Ubuntu. It seems its working fine in Linux.

**Comment by jayrawatrj** [comment]
No, i can ooen julytrr and work on that but spyder is not working. I have
tried almost all possible way to solve it but nothing worked.

**Comment by Suphawit01** [comment]
I also have the same problem :(

**Comment by zcq0730** [comment]
This works for me: In windows 10 (run this in conda prompt, might need to run as admin)

conda install --force-reinstall pyqt qt

see here:
spyder-ide/spyder#6934

**Comment by zcq0730** [comment]
try this: 

This works for me: In windows 10 (run this in conda prompt, might need to run as admin)

conda install --force-reinstall pyqt qt

see here:
spyder-ide/spyder#6934

**Comment by ammubharatram** [comment]
> spyder --reset
> solves the problem

The above, after reinstalling pyqt (quoted below) solves the problem for me.

**Comment by JRonGitHub** [comment]
I had the following error message, when trying to launch Spyder from my conda environment (in other words the environment was already activated and running)

This Python interpreter is in a conda environment, but the environment hasnot been activated. Libraries may fail to load. To activate this environmentplease see https://conda.io/activation


The above Spyder --reset did the trick for me (I have also used 'update qt' though before that as suggested on one of the stackoverflow answers... just before that, so maybe try both)

**Comment by JustinNewport** [comment]
I have Windows 10 and Spyder wont even launch for the first time immediately after installing the most recent version of Anaconda. Anyone else having this issue recently? (November 2020)

**Comment by foobar77** [comment]
I have **Windows 7** and Spyder wont even launch for the first time immediately after installing the most recent version of Anaconda.  Next I tried installing WinPython63 3.8.6.0 in order to launch spyder from there. No luck there too.

**Comment by ccordoba12** [comment]
@stefaneidelloth, it seems you're trying to use a version older than Spyder 3.3.0 with Python 3.10. But that version was released three and a half years ago.

**Comment by gersbrett** [comment]
I had the same problem with a download of Winpython: spyder would just not start without any message. It turned out that even I had the spyder.exe the python modules were missing. So pip install spyder solved the issue

**Comment by Hagtronics** [comment]
@gersbrett - Agreed, the latest windows installer is broken and will not launch Spyder properly. Installing Spyder from pip instead fixed the startup issue for me on 22H2 Win10. Thanks for the solution.

**Comment by Hagtronics** [comment]
@ccordoba12 - I wasn't trying to be overly critical, Windows is a convoluted environment to be sure, and I am not running WinPython. I have just had issues getting Spyder to run properly over the last three releases, whereas it ran without a hitch before that (January 2022 time frame). I run a straight Python 3.9 install on Win10, no environments, no Anaconda, etc. I have found that Spyder when I try your install and run the shortcut your installer provides, try's to launch but then quits at the splashscreen. When I coax it to tell me what's wrong, it tells me it can't find Python 3.8. So then I have had to write a batch file that launches Python 3.8 from your windows installer install and then launches the spyder_xxx.pyw file you provide. 

When I installed Spyder via PIP (@gersbretts hint above), then Spyder.exe was placed properly in my main Python 3.9 directory(s) like everything else, and all I have to do to get this to run properly is launch it as an administrator.

**Comment by ccordoba12** [comment]
> @ccordoba12 - I wasn't trying to be overly critical, Windows is a convoluted environment to be sure, and I am not running WinPython

Don't worry, it wasn't clear from your message if you were using WinPython. I read it that way, hence my suggestion to report the problem to them.

> I have just had issues getting Spyder to run properly over the last three releases, whereas it ran without a hitch before that (January 2022 time frame)

Ok, this could be due to the inclusion of Spyder-terminal in our Windows installer. You could check that by going to the `site-packages` directory of the installer (I this it's called `pkgs`) and removing the folder for Spyder-terminal.

> So then I have had to write a batch file that launches Python 3.8 from your windows installer install and then launches the spyder_xxx.pyw file you provide.

Could you post the error you're getting and the script you're using to launch the installer? That way we could better understand your problem.

**Comment by Hagtronics** [comment]
@ccordoba12 - OK, I installed the full installer and looked at this again.

I removed the terminal - that didn't fix the issue.

If I just run the installer supplied shortcut, I get a splash screen, some things zip by and then the splash screen disappears. Poof - Gone.

If I run the installer supplied shortcut as an administrator - I get a splash screen and then I "Think" I can see the info zip by that it is changing directory's, etc. Finally Spyder opens fine.

I am 99% sure that there is some permissions problem (you know, because Windows 10 is a nightmare with permissions). I take back what I said about the installer - this is some interaction with Win 10 and permissions. 

The only weird thing is that in January / February 22 this worked fine with the installer out of the box, but there have been monthly Win10 updates, so perhaps there is an interaction there.

**Bottom line: Just run the shortcut as an administrator and it works.**

Thanks for your help and prodding so I got to the bottom of it.

**Comment by ccordoba12** [comment]
> The only weird thing is that in January / February 22 this worked fine with the installer out of the box, but there have been monthly Win10 updates, so perhaps there is an interaction there.

Agreed, that's very strange. We did many changes between Spyder 5.2.2 (release on January) and 5.3.0 (released on March), so it'd be hard to tell exactly what is causing this issue.

> Thanks for your help and prodding so I got to the bottom of it.

No problem. Glad to know you found a workaround to keep using Spyder.

> I should add, that I don't see a lot of chatter here with others having these same sorts of issues that I have, so it seems to be more of a one off on my configuration problem.

Yep, that's my suspicion too.
